src/canmatrix/canmatrix.py

```diff
@@ -574,10 +617,10 @@ class Frame(object):
     """
 
     name = attr.ib(default="")
-    id = attr.ib(type=int, default=0)
+    arbitration_id = attr.ib(converter=Arbitration_Id.from_compound_integer, default=0)
```

Comment:
Do we need to be so verbose re: `arbitration_id` over `id`?  It might be better, but I'm not sure there's likely much confusion with just `id`.  Either way.

Reply:
Attention, `id` is a python keyword. It would lead to "shadowing" warning.

Reply:
As it presumably has been already.  I can't say I like it but it only shadows in the class definition itself.  Once you are outside the class definition there's no more shadowing since it's `self.id`.  Since you mostly shouldn't use `id()` at all anyways...  I think I'm ok with this.  It's a hazard of mixing declarative with imperative like attrs does in this style usage.